CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next Release
 
 1. (New) Make `--order project` work. ([Issue #49](https://github.com/roberts1000/rspec_n/issues/49))
+1. (New) Add folder output options. ([Issue #455](https://github.com/roberts1000/rspec_n/issues/55))
```

Comment:
Typo here, I think you meant Issue #55 inside the [ ]

Reply:
Thanks for the heads-up.